xml/decoders/SoundTraxx_Tsu_Diesel_Bachmann_SndVal.xml

```diff
@@ -55,6 +55,8 @@
                     from SoundTraxx Product Identification guide Rev.B                                  -->
   <!-- version 10  	Add ditch light support FX5/FX6 to GE_ES44AC, EMD_SD70ACe as reported by User 
 					Add EMD E7, EMD GP30																-->
+  <!-- version 11  	Remove GG1 Electric, moved to its own definition. Few corrections as per User feedback 
+					regarding CV's      																-->
```

Comment:
I might be missing something here. Decoder files are not my expertise. But you write that you have moved GG1 Electric to its own definition. Would that mean a separate decoder file? If so, I don't see it in this PR. Or does it come later?

@bobjacobsen plans to build the next test release Tuesday evening. I have approved this PR and I'm happy to merge it, but my question is if that should wait until GG1 Electric is added, if it's added to a separate file?

Reply:
The GG1 is already  in JMRI in the Bachmann SoundValue Electric definition.   The GG1 was simply never removed from this definition where is should not have been.

Reply:
I see. Thanks for the explanation.